recipe/migrations/netcdf-fortran-4.5.x.yaml

```diff
@@ -7,5 +7,5 @@ __migrator:
   build_number:
     1
 
-netcdf-fortran:
+netcdf_fortran:
```

Comment:
Thanks. I actually explained that in your PR #341 but totally missed it here. (It is hard to read `-` vs `_`!)